### PR TITLE
use external-secrets for pyxis

### PR DESCRIPTION
**Pull request by scoheb**
- pull in pyxis secret from vault using external-secrets
- refer to new secret in release-strategy

### CODE REVIEW

File: ci/teams/rhtap-servicerelease/external-secrets/pyxis-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+kind: ExternalSecret
+metadata:
+  name: pyxis-staging-secret
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        key: "staging/release/pyxis/certificate"
```
Comment (johnbieren) [inline ci/teams/rhtap-servicerelease/external-secrets/pyxis-secret.yaml:11]: This is a directory. This works? So it injects both the key and certificate and knows to decode them?

Comment (johnbieren) [reply ci/teams/rhtap-servicerelease/external-secrets/pyxis-secret.yaml:11]: okay cool

**Review by johnbieren — APPROVED** [review]

**Comment by scoheb** [comment]
another non-related test failure

**Pull request merged**